server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/QueryScheduleTaskParams.java

```diff
@@ -32,7 +32,7 @@ public class QueryScheduleTaskParams {
     private List<Schedule> schedules;
     private String scheduleName;
     private Set<Long> dataSourceIds;
-    private List<TaskStatus> statuses;
+    private TaskStatus status;
```

Comment:
status in can cover status equals scenario, may not neccessary to change it ?

Reply:
Changed back